src/helpers/local-storage.js

```diff
@@ -0,0 +1,48 @@
+export const ErrorSafeLocalStorage = {
+    setItem: (key, value) => {
+      try {
+        localStorage.setItem(key, value);
+      } catch {
+        return;
+      }
+    },
+    getItem: (key) => {
+      try {
+        return localStorage.getItem(key);
+      } catch {
+        return null;
+      }
+    },
+    removeItem: (key) => {
+      try {
+        localStorage.removeItem(key);
+      } catch {
+        return;
+      }
+    },
+  };
+  
+  export const ErrorSafeSessionStorage = {
```

Comment:
What do you think about moving session storage to another helper file named `session-storage.js`?

Reply:
Fair enough